internal/display/tenants.go

```diff
@@ -1,25 +1,32 @@
 package display
 
 type tenantView struct {
-	Name string
-	raw  interface{}
+	Active bool
+	Name   string
+	raw    interface{}
 }
 
 func (v *tenantView) AsTableHeader() []string {
-	return []string{"Available tenants"}
+	return []string{"Active", "Tenant"}
 }
 
 func (v *tenantView) AsTableRow() []string {
-	return []string{v.Name}
+	activeText := ""
+	if v.Active {
+		activeText = boolean(v.Active)
```

Comment:
I feel like an arrow would make more sense rather than a check icon.
```suggestion
		activeText = ansi.Green("▸")
```